plaso/engine/processing_status.py

```diff
@@ -508,6 +518,19 @@ def UpdateWorkerStatus(
         number_of_consumed_reports, number_of_produced_reports)
 
 
+class EventsStatus(object):
+  """The status of the events.
```

Comment:
Please confirm you have plans to add more attributes to this class. Otherwise, we can just use an int and not add another layer of indirection.

Reply:
Yes I think we could add other information such as:

* Events MACB grouped
* Duplicate events removed
* Events filtered